packages/client/src/views/Performance/FieldAgentList.tsx

```diff
@@ -193,7 +193,7 @@ function FieldAgentListComponent(props: IProps) {
         timeEnd: timeEnd,
         locationId: locationId,
         status: status.toString(),
-        event: event || undefined,
+        event: event?.toUpperCase() || undefined,
```

Comment:
If we give a default value to event then we don't need to pass undefined here anymore